server/routes/message.js

```diff
@@ -0,0 +1,11 @@
+const express = require('express');
+const router = express.Router();
+
+const protect = require('../middleware/auth');
+const {getMessages, createMessage} = require('../controllers/message');
+
+
+router.route("/").get(protect, getMessages);
```

Comment:
use `router.route("/:conversationId").get(protect, getMessages);` to get req params as stated above

Reply:
Updated accordingly and resolved undefined errors as well. Thanks!